RomeALincei/ANLcr53.xml

```diff
@@ -0,0 +1,394 @@
+                                            ቀትል<supplied reason="omitted">፡</supplied></seg>
+                                    </q>
+                                </item>
+                                
+                                <item xml:id="e1">
+                                    <desc>The name of the owner was <persName ref="PRS14732WaldaMB" role="owner"/>
+                                        (cp. <locus target="#3r #83v"/>) and that of the scribe <persName ref="PRS14733HabtaM" role="scribe"/>
+                                    (<locus target="#81v #83v"/>).</desc>
+                                </item>
+                                
+                                <item xml:id="e2">
+                                    <locus target="#86v"/>
+                                    <desc>Scribbles in pencil.</desc>
+                                </item>
+                                
+                                <item xml:id="e3">
+                                    <locus target="#2v"/>
+                                    <desc type="StampExlibris">Ex libris <persName ref="PRS3182ContiRo">CCR</persName></desc>
+                                </item>
+                            </list>
+                        </additions>
+                        
+                        <bindingDesc>
+                            <binding>
+                                <decoNote xml:id="b1">Wooden boards.</decoNote>
+                                <decoNote xml:id="b2" type="bindingMaterial">
+                                    <material key="wood"/>
+                                </decoNote>
+                                <decoNote xml:id="b3" type="SlipCase">Kept in a
+                                    leather <foreign xml:lang="gez">māḫdar</foreign>.</decoNote>
+                            </binding>
+                        </bindingDesc>
+                    </physDesc>
+                    
+                    <history>
+                        <origin>
+                           The catalogue dates the production of the manuscript to the
+                            <origDate notBefore="1800" notAfter="1899">19th century</origDate>.
+                        </origin>
+                        <provenance>The manuscript was either acquired by <persName ref="PRS3182ContiRo" role="owner">Carlo Conti Rossini</persName> 
+                            during his stay in <placeName ref="LOC2991Eritre">Eritrea</placeName>
+                            (<date notBefore="1899" notAfter="1903">1899–1903</date>) or sent to him after his return to Europe.</provenance>
+                        <acquisition>The manuscript passed into the possession of the library in <date when="1950">1950</date>, following Conti Rossini’s death in <date when="1949">1949</date> 
+                            (see <ref target="https://www.menestrel.fr/?-Roma-&amp;lang=fr#845">Menestrel</ref>).</acquisition>
```

Comment:
Could this also be listed as a catalogue in the bibliography, if you used it for the description? Or is it unnecessary?

Reply:
I think so, it is a general overview of the collection, not a description of the manuscript in question